src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -77,6 +79,27 @@ public function iAddOfThemToMyCart(int $quantity, ProductInterface $product, str
         $this->putProductToCart($product, $tokenValue, $quantity);
     }
 
+    /**
+     * @When /^I remove (product "[^"]+") from the (cart)$/
+     */
+    public function iRemoveProductFromTheCart(ProductInterface $product, string $tokenValue): void
+    {
+        $items = $this->responseChecker->getValue($this->cartsClient->show($tokenValue), 'items');
+
+        foreach ($items as $item) {
+            $pathElements = explode('/', $item['variant']['product']);
+
+            $productVariantCode = $pathElements[array_key_last($pathElements)];
```

Comment:
```suggestion
            $productCode = $pathElements[array_key_last($pathElements)];
```
However, I would say, that just making a request for product would be a better solution.